packages/sanity/src/structure/structureBuilder/Component.ts

```diff
@@ -205,6 +208,10 @@ export class ComponentBuilder implements Serializable<Component> {
     return this.spec.menuItemGroups
   }
 
+  canHandleIntent(canHandleIntent: any): ComponentBuilder {
```

Comment:
I think the `canHandleIntent` parameter should be typed as [`IntentChecker`](https://github.com/sanity-io/sanity/blob/9c86166ee63064d3a2c714d8a89a380ed668f9bc/packages/sanity/src/structure/structureBuilder/Intent.ts#L85-L96)?

```suggestion
  canHandleIntent(canHandleIntent: IntentChecker): ComponentBuilder {
```

Reply:
🤦 thank you! too hasty to commit